### PR TITLE
Add attachments option to user verification email

### PR DESCRIPTION
**Pull request by reichert621**
This PR allows an array of `attachments` to be sent in the `sendEmail` function of `User#verify`.

Reference: https://github.com/andris9/Nodemailer#attachments

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: common/models/user.js [hunk 1]
```diff
@@ -372,7 +372,8 @@ module.exports = function(User) {
         subject: options.subject || 'Thanks for Registering',
         text: options.text,
         html: template(options),
-        headers: options.headers || {}
+        headers: options.headers || {},
+        attachments: options.attachments || []
```
Comment (ritch) [inline common/models/user.js]: This should be:

```
attachments: options.attachments || undefined
```

**Comment by ritch** [comment]
I'm closing this PR... If you make the requested change ping me (via @ritch) and we'll reopen and merge.

**Comment by ritch** [comment]
@reichert621

**Pull request closed without merging**